LocalSettings.php

```diff
@@ -1507,6 +1507,40 @@
 			'mode' => 'packed',
 		],
 	],
+	
+	// GeoData
+	'wgGlobes' => [
+		'default' => [],
+		'gratisdatawiki' => [
+			'https://gratisdata.miraheze.org/entity/Q476' => 'earth',
```

Comment:
Where did you find documentation for this? I can't, and I'm not sure it supports Wikibase entities without first knowing that.